2026-T2/BDA/modules/notes.md

```
@@ -3,4 +3,102 @@
+#### 2. Interactive Knowledge Check
+
+Please click the link below to complete the Module 1 Knowledge Check.
+You can attempt this as many times as you like.
+
+https://a19788-10605129.cluster228.canvas-user-content.com/courses/19788~26571/files/19788~10605129/course%20files/htmlResources/content/index.html?download=1&inline=1&sf_verifier=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJ1c2VyX2lkIjoiMTk3ODgwMDAwMDAwMDg2Mjg3IiwiYXR0YWNobWVudF9pZCI6IjE5Nzg4MDAwMDAxMDYwNTEyOSIsInBlcm1pc3Npb24iOlsicmVhZCIsImRvd25sb2FkIl0sInJvb3RfYWNjb3VudF9pZCI6IjE5Nzg4MDAwMDAwMDAwMDAwMSIsIm9hdXRoX2hvc3QiOiJteWxlYXJuLnRvcnJlbnMuZWR1LmF1IiwicmV0dXJuX3VybCI6bnVsbCwiZmFsbGJhY2tfdXJsIjoiaHR0cHM6Ly9teWxlYXJuLnRvcnJlbnMuZWR1LmF1L2NvdXJzZXMvMjY1NzEvZmlsZXMvMTA2MDUxMjkvcHJldmlldz9mYWxsYmFja190cz0xNzc5ODcxNjk5IiwianRpIjoiMzRjN2MzNzktMWMwYS00M2Q1LWEzMGYtNzE2YjJlMjhhN2MyIiwiZXhwIjoxNzc5ODcxOTk5fQ.koifHZ1nBp4IkXQGoj4_a4i5-jgQGaRbISNL5S3NG7PgM22cqcim1u9NDcsL3dJj-dl3EtTtIJMT3qY2GAoKhQ#/
+
+> *Status: 🕐 To-Do*
+
+---
+
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)